docs/guides/getting-started/setup/sveltekit.mdx

```diff
@@ -0,0 +1,218 @@
+```
+
+  </TabItem>
+
+  <TabItem value="pnpm">
+
+```shell
+pnpm install -D @sveltejs/adapter-static@next
+```
+
+  </TabItem>
+</Tabs>
+
+Then update the `adapter` import in the `svelte.config.js` file:
+
+```javascript title=svelte.config.js
+import adapter from '@sveltejs/adapter-static' // This was changed to adapter-static!
+import preprocess from 'svelte-preprocess'
+
+/** @type {import('@sveltejs/kit').Config} */
+const config = {
+  // Consult https://github.com/sveltejs/svelte-preprocess
+  // for more information about preprocessors
+  preprocess: preprocess(),
+
+  kit: {
+    adapter: adapter(),
+  },
+}
+
+export default config
+```
+
+Lastly we need to disable SSR and enable prerendering by adding a root `+layout.ts` file (or `+layout.js` if you are not using TypeScript) with these contents:
+
+```typescript title=src/routes/+layout.ts
+export const prerender = true
+export const ssr = false
+```
+
+Note that static-adapter doesn't require you to disable ssr for the whole app but it makes it possible to use APIs which depend on the global `window` object, like Tauri's API, without [Client-side checks].
+
+Furthermore, if you prefer a Single-Page Application (SPA) mode over SSG, you can change the adapter configurations and `+layout.ts` according to the [adapter docs].
+
+## Create the Rust Project
+
+<TauriInit
+  destDirExplination={{
+    __html: 'Use <code>../build</code> for this value.',
+  }}
+  devPathExplination={{
+    __html: 'Use <code>http://localhost:5173</code> for this value.',
+  }}
+  beforeDevCommandExplination={{
+    __html:
+      'Use <code>npm run dev</code> (be sure to adapt to use your package manager of choice).',
+  }}
+  beforeBuildCommandExplination={{
+    __html:
+      'Use <code>npm run build</code> (be sure to adapt to use your package manager of choice).',
+  }}
+/>
+
+Now that we have scaffolded our frontend and initialized the Rust project the created `tauri.conf.json` file should look like this:
+
+```json title=src-tauri/tauri.conf.json
+{
+  "build": {
+    // This command will execute when you run `tauri build`.
+    "beforeBuildCommand": "npm run build",
+    // This command will execute when you run `tauri dev`
+    "beforeDevCommand": "npm run dev",
+    "devPath": "http://localhost:5173",
+    "distDir": "../build"
+  },
+```
+
+And that's it! Now you can run the following command in your terminal to start a development build of your app:
+
+<Command name="dev" />
+
+![Application Window](./assets/sveltekit/sveltekit-dev.png)
+
+## Invoke Commands
+
+<Commands />
+
+To call our newly created command we will use the [`@tauri-apps/api`] JavaScript library. It provides access to core functionality such as windows, the filesystem, and more through convenient JavaScript abstractions. You can install it using your favorite JavaScript package manager:
+
+<InstallTauriApi />
+
+With the library installed, you can modify your `+page.svelte` file to call the Command:
```

Comment:
```suggestion
With the library installed, you can modify your `+page.svelte` file to call the command:
```

Reply:
i copied that from some other guide i think so we may need to look through them too 😂